movieApp.js

```diff
@@ -1,204 +1,279 @@
+      movieDetails.appendChild(commentSection);
+    }
+    createCommentSection();
 
-    // append
-    commentSection.appendChild(commentInput);
-    commentSection.appendChild(commentButton);
-    commentSection.appendChild(comments);
-    movieDetails.appendChild(commentSection);
-  }
-  createCommentSection();
-
-  function toggle() {
-    {
-      this.classList.toggle("active");
-      /* Toggle between hiding and showing the active panel */
-      if (movieDetails.style.display === "block") {
-        movieDetails.style.display = "none";
-      } else {
-        movieDetails.style.display = "block";
+    function toggleMovie() {
+      {
+        this.classList.toggle("active");
+        /* Toggle between hiding and showing the active panel */
+        if (movieDetails.style.display === "block") {
+          movieDetails.style.display = "none";
+          movieDataContainer.style.display = "inline-block";
+        } else {
+          movieDetails.style.display = "block";
+          movieDataContainer.style.display = "flex";
+        }
       }
     }
+
+    readMoreBtn.addEventListener("click", toggleMovie);
+    img.addEventListener("click", toggleMovie);
+    return movieDataContainer;
   }
 
-  readMoreBtn.addEventListener("click", toggle);
-  return movieContainer;
-}
+  function createMoviesPage() {
+    nowStreaming.innerHTML = "";
+    movies.forEach((movie) => {
+      const movieElement = createMovieCard(movie);
+      nowStreaming.appendChild(movieElement);
+    });
+  }
+  createMoviesPage();
+  // JS 2
+  const searchBtn = document.getElementById("search-icon");
 
-function createMoviesPage() {
-  nowStreaming.innerHTML = "";
-  movies.forEach((movie) => {
-    const movieElement = createMovieCard(movie);
-    nowStreaming.appendChild(movieElement);
-  });
-}
-createMoviesPage();
-// JS 2
-const searchBtn = document.getElementById("search-icon");
-
-// function to search
-const inputField = document.getElementById("keyword-search");
-searchBtn.addEventListener("click", () => {
-  const inputStr = inputField.value.toLowerCase();
-  nowStreaming.innerHTML = "";
-  let found = false; // Flag to check if any movie is found
-
-  movies.forEach((movie) => {
-    if (
-      movie.title.toLowerCase().includes(inputStr) ||
-      movie.description.toLowerCase().includes(inputStr)
-    ) {
-      found = true;
-      const searchMovieElement = createMovieCard(movie);
-      nowStreaming.appendChild(searchMovieElement);
-    }
+  // function to search
+  const inputField = document.getElementById("keyword-search");
+  searchBtn.addEventListener("click", () => {
+    const inputStr = inputField.value.toLowerCase();
+    nowStreaming.innerHTML = "";
+    let found = false; // Flag to check if any movie is found
+
+    movies.forEach((movie) => {
+      if (
+        movie.title.toLowerCase().includes(inputStr) ||
+        movie.description.toLowerCase().includes(inputStr)
+      ) {
+        found = true;
```

Comment:
When the movie is found, the code will continue iterating the rest of the elements, even though it is not necessary. Can you think of a way to avoid that? (hint: it's just a single line and a single word)